docs/usage.md

```diff
@@ -115,8 +219,45 @@ Calculates a weighted ratio based on the other ratio algorithms.
 ## process
 
 ### extract
+
 Find the best matches in a list of choices.
 
+Parameters:
+
+- **query**: *str*
+
+    String we want to find.
+
+- **choices**: *Iterable*
+
+    List of all strings the query should be compared with or dict with a mapping `{<result>: <string to compare>}`
+
```

Comment:
I think we should mention, that a mapping is anything that provides a `items` method, which can not only be a dictionary, but e.g. a pandas.Series aswell (index: element).